microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,93 @@
+
+Users want to see dates, times, numbers, and other formatting printed according to their preferences. For example, the US geographic region translates **7/2/2022** into **July 2nd, 2022**.  Another geographic region could translate that same date into **February 7, 2022** because they read the month and day inversely. 
+
+## Regional Codes
+
+The user's formatting preferences are abstracted into a simple regional code, which is used by web applications to infer the various details noted above about dates, times, and other formatting preferences.
+
+By using a regional code, websites can use the built-in Internationalization API to properly format information into the user's preferences.  The Internationalization API saves lots of MB of data transfer over the network, keeping websites small and allowing them to quickly and easily adapt to different regions.
+
+In general, the website's internationalization customer experiences are only as good as the regional code that they're able to determine for the current user. 
+
+A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU, which is an implementation of the Unicode standard's Internationalization API.  See [ICU](https://icu.unicode.org/home).
+
+To set the user's language preferences, Microsoft Edge gets the default values from the OS at the time it's first installed. After that, users can change the language preferences on **edge://settings/languages** and then sorting them in priority order.
+
+|Microsoft Edge Languages settings| Description |
+|--------|-------------------------|
+| **Microsoft Edge is displayed in this language** | One item in the list of **Preferred languages** indicates the language used for rendering Microsoft Edge's UI|
+| The Language used to render web sites | **Website Server-side processing**: Microsoft Edge sends the language list (as regional codes) specified in **edge://settings/languages** via the `accept-language` header. <br><br> **Website Client-side processing**: Microsoft Edge makes the language list (as regional codes) available via the `navigator.languages` DOM API. The preferred language (at the top of the list) is also available as `navigator.language`. |
+| The language/region used for formatting values such as date, time, and numbers. | The language in the `edge://settings/languages` list specified as the **displayed language**. |
+
+#### Regional format
+To reflect a customer's regional preferences (especially when the user has changed their default preferences in the OS), Microsoft Edge has a new setting in the **Language** section of **Settings**:
+
+![Share additional OS regional format with the option **limited** selected.](media/os-regional-setting-screenshot.msft.png)
+
```

Comment:
@travisleithead - can you get the UI label changed to add hyphen before the two occurrences of "based"?

Reply:
It's possible, but a big hassle and will take a while to propogate to Stable channels, since it's a product string change. If we can avoid changing it that would be my preference.